tests/basic/kubevirt_test.go

```diff
@@ -827,3 +830,86 @@ var _ = Describe("{KubeVirtPvcAndPoolExpandWithAutopilot}", func() {
 		}
 	})
 })
+
+var _ = Describe("{RebootRootDiskAttachedNode}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootRootDiskAttachedNode", "Reboot the node where VMs root disk is attached", nil, 0)
+		DeployVMTemplatesAndValidate()
+	})
+	var appCtxs []*scheduler.Context
+
+	itLog := "Reboot node where Kubevirt VMs root disk is attached"
+	It(itLog, func() {
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		Inst().AppList = []string{"kubevirt-cirros-live-migration", "kubevirt-windows-vm"}
+		stepLog := "schedule a kubevirtVM"
+		Step(stepLog, func() {
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				log.Infof("RK=> Deploying iteration: %d", i+1)
+				appCtxs = append(appCtxs, ScheduleApplications("reboot")...)
+			}
+		})
+		//defer DestroyApps(appCtxs, nil)
+		ValidateApplications(appCtxs)
+		for _, appCtx := range appCtxs {
+			bindMount, err := IsVMBindMounted(appCtx, false)
+			log.FailOnError(err, "Failed to verify bind mount")
+			dash.VerifyFatal(bindMount, true, "Failed to verify bind mount")
+		}
+
+		stepLog = "Get node where VM's root disk is attached and reboot that node"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+
+			for _, virtualMachineCtx := range appCtxs {
+				bindMount, err := IsVMBindMounted(virtualMachineCtx, false)
+				log.FailOnError(err, "Failed to verify bind mount pre node reboot")
+				dash.VerifyFatal(bindMount, true, "Failed to verify bind mount pre node reboot")
+
+				vms, err := GetAllVMsFromScheduledContexts([]*scheduler.Context{virtualMachineCtx})
+				log.FailOnError(err, "Failed to get VMs from scheduled contexts")
+				dash.VerifyFatal(err, nil, "Failed to to get VMs from scheduled contexts")
+
+				for _, vm := range vms {
+					nodeName, err := GetNodeOfVM(vm)
+					log.FailOnError(err, "Failed to get node name for VM: %s", vm.Name)
+					log.Infof("Pre-reboot VM [%s] in namespace [%s] is scheduled on node [%s]. Rebooting it.", vm.Name, vm.Namespace, nodeName)
+					nodeObj, err := node.GetNodeByName(nodeName)
+					log.FailOnError(err, "Failed to get node obj for node name: %s", nodeName)
+
+					err = Inst().N.RebootNodeAndWait(nodeObj)
+					log.FailOnError(err, "Failed to reboot  node: %s", nodeObj.Name)
+					log.Infof("Succesfully rebooted node: %s", nodeObj.Name)
+				}
+
+				ValidateApplications(appCtxs)
+
+				// Get updated VM list and validate bind mount again
+				vms, err = GetAllVMsFromScheduledContexts([]*scheduler.Context{virtualMachineCtx})
+				log.FailOnError(err, "Failed to get VMs from scheduled contexts")
+				dash.VerifyFatal(err, nil, "Failed to to get VMs from scheduled contexts")
+
+				for _, vm := range vms {
+					nodeName, err := GetNodeOfVM(vm)
+					log.FailOnError(err, "Failed to get node name for VM: %s", vm.Name)
+					log.Infof("Post reboot VM [%s] in namespace [%s] is scheduled on node [%s]", vm.Name, vm.Namespace, nodeName)
+				}
+
+				bindMount, err = IsVMBindMounted(virtualMachineCtx, false)
```

Comment:
check if the VM is ready as well

Reply:
`ValidateApplications` on line 986 should take care of it.